metrics-datadog/src/test/java/org/coursera/metrics/datadog/TaggedNameTest.java

```diff
@@ -1,10 +1,10 @@
 package org.coursera.metrics.datadog;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import static org.coursera.metrics.datadog.TaggedName.TaggedNameBuilder;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I thought we are "against" asterisk imports.

Reply:
Oops -- my intention was to keep wildcard imports where they were, but not expand their use. I didn't intend to change the coding standards for this project. I'll replace this wildcard import but leave the other.